fix(client): add bottom spacer to ToolsSidebar so last item can scroll into view

Fikri-20 opened the pull request:
I Inserted a bottom spacer element after the tools list: a <div className="h-10" aria-hidden /> inside the grid that renders [filteredToolNames] 
I Inserted the same spacer after the saved-requests list (inside the space-y-1 wrapper) as well. 
<img width="1884" height="1448" alt="screenshot-2025-10-17_19-35-22" src="https://github.com/user-attachments/assets/31bd084b-f23f-4c97-babb-164e303d2709" />

Fikri-20 commented:
I've formatted the code, as well as lint it.
Please recheck tests

ahmedfikri555-spec committed:
fixing the scrollbar && resolved TypeScript Lint Errors

Fikri-20 commented:
I fix an error occured in the Resources page. 
Now Tools, Resources and Prompts are done, scrollbar is adjusted properly 
<img width="1403" height="1449" alt="screenshot-2025-10-18_02-33-38" src="https://github.com/user-attachments/assets/3dd932c5-1470-491f-a024-7019df026d08" />
<img width="1404" height="1454" alt="screenshot-2025-10-18_02-33-06" src="https://github.com/user-attachments/assets/5aa4a9dd-0c11-4933-9b6c-80de6bc556a9" />

please check the tests again.

ahmedfikri555-spec committed:
chore: remove cached files and node_modules from git

chelojimenez commented:
Hi @Fikri-20! I don't see your changes? 
<img width="1104" height="385" alt="image" src="https://github.com/user-attachments/assets/dbafd448-cb6e-4b27-87ff-912affd37015" />

ahmedfikri555-spec committed:
ignoring node module from pushing

ahmedfikri555-spec committed:
removing .npm-cache && ignoring it

Fikri-20 committed:
Merge branch 'main' into feature/f1

Signed-off-by: Ahmed Fikri <149339473+Fikri-20@users.noreply.github.com>

Fikri-20 commented:
Hi @chelojimenez,

Apologies — I accidentally pushed the build binaries earlier.
The .npm-cache/ folder wasn’t included in the .gitignore file, so I’ve now updated it. I also removed all unnecessary binaries and node modules from the commit, keeping only the relevant changes for this issue.

Please review again, and sorry for the inconvenience.

ahmedfikri555-spec committed:
removing lint changes & @ts-ignore

ahmedfikri555-spec committed:
Merge branch 'feature/f0' of https://github.com/Fikri-20/inspector into feature/f1

ahmedfikri555-spec committed:
cont. removing lint changes
